src/components/HistoryPanel.tsx

```diff
@@ -72,23 +88,41 @@ export default function VotePanel(_props: Props) {
       q,
       outcomeByHandle.get(q.questionHandle),
     ]);
-  const receivedItems = questionsWithAnswers.map(([qData, aData], index) => (
-    <motion.div
-      key={index}
-      variants={cardVariant}
-      className="rounded-lg border-gray border shadow-md mb-4"
-    >
-      <QuestionDetails
-        details={qData}
-        outcome={aData?.question === qData.questionHandle ? aData : undefined}
-        instance={instance}
-      />
-    </motion.div>
-  ));
+  const receivedItems =
+    instanceStatus === 'received' && questionsWithAnswers.length ? (
+      questionsWithAnswers.map(([qData, aData], index) => (
+        <motion.div
+          key={index}
+          variants={cardVariant}
+          className="rounded-lg border-gray border shadow-md mb-4"
+        >
+          <QuestionDetails
+            details={qData}
+            outcome={
+              aData?.question === qData.questionHandle ? aData : undefined
+            }
```

Comment:
these come from the same tuple so why wouldn't they match? If you're just handling no outcome, can QuestionDetails be responsible for that?

```suggestion
            outcome={aData}
```